test/TestAsset/http-status-codes.xml

```diff
@@ -0,0 +1,368 @@
+<?xml version='1.0' encoding='UTF-8'?>
```

Comment:
OK. I understand.

Check new version, I may remove "last modified" stuff to make it more simpler.

Apparently IANA server don't support "If-Modified-Since" (or I made some mistakes).

But. I think what timeout and fallback to file is necessary.